random_change_color.py

```diff
@@ -0,0 +1,12 @@
+import cv2
+from random import randint
+
+
+def random_change_color(img):
```

Comment:
I'm not sure, but maybe it would be better to name variables and functions without the "_" symbol